src/language/pythonLanguage.ts

```diff
@@ -100,6 +101,41 @@ export const pythonLanguage: Language = {
       )
       (#match? @method "(given|when|then|step)")
     ) @root`,
+    // pytest-bdd
+    `(decorator
+      (call
+        function: (identifier) @annotation-name
```

Comment:
```suggestion
        function: [(identifier)(attribute)] @annotation-name
```
in case the parser is called as an attribute of a module